app/assets/stylesheets/local/typography.scss

```diff
@@ -90,3 +90,8 @@ address {
   text-decoration: underline;
   margin-bottom: 0;
 }
+
+// TODO: add iconography
```

Comment:
@axemonkey Are you able to add the red exclamation mark to this PR, before we merge it? I'm worried we might forget, otherwise.